### ISSUE TITLE
Import button doesn't appear

### ISSUE THREAD

**Issue by PierreMesure** [digidemlab/decidim-module-budgets_paper_ballots#1]
In my [original PR](https://github.com/decidim/decidim/pull/8680), I used the import classes created by @lahdeero to create a standard import form for paper ballot results.

I declare a new type of import in the component's [manifest](https://github.com/digidemlab/decidim-module-budgets_paper_ballots/blob/fec163a86f01184139704f5fef6c7059fff3aa63/lib/decidim/budgets/component.rb#L78) and I try to display a button to access the import view using [this](https://github.com/digidemlab/decidim-module-budgets_paper_ballots/blob/fec163a86f01184139704f5fef6c7059fff3aa63/app/views/decidim/budgets/admin/projects/index.html.erb#L10), the same code used for [proposal answer import](https://github.com/decidim/decidim/blob/a83cfd096eaec0fbe6c7d9ea4aac86018a459ee4/decidim-proposals/app/views/decidim/proposals/admin/proposals/_bulk-actions.html.erb#L18). This works perfectly in the PR, but it doesn't when I isolate the logic in this module. The button doesn't appear and the debugger tells me it's because in my component's manifest, `import_manifests=[]`.

The module only contains the modified files from the PR so it's gross monkey-patching, no other changes.

@lahdeero, do you have any idea why my import manifest is empty despite monkey-patching the component.rb file?

Thanks in advance!

**Comment by lahdeero** [comment]
> @lahdeero, do you have any idea why my import manifest is empty despite monkey-patching the component.rb file?

It's hard to say without debugging, but note that imports works slightly different in 0.25 than in develop. For example 0.25 didn't have messages block.

First what I would check if the manifest is correctly created.

**Comment by PierreMesure** [comment]
Thank you @lahdeero! This morning, @microstudi pointed me to a [bit of code](https://github.com/AjuntamentdeBarcelona/decidim-barcelona/blob/57e96dfffb5a897ad1534f979bd43c6d1e3687fe/decidim-ephemeral_participation/lib/decidim/ephemeral_participation/engine.rb#L32) doing exactly that in a module they created for Barcelona.

I’ll try next week!

**Comment by PierreMesure** [comment]
Thank you @microstudi and @lahdeero! I managed to fix it and learnt a lot while doing it!

The solution was to inject the import logic in the component's manifest [like this](https://github.com/digidemlab/decidim-module-budgets_paper_ballots/blob/3e5d2794f43b1c353e8a3116edf6daab9e3a26a7/lib/decidim/budgets_paper_ballots/engine.rb#L22).

And regarding the differences between 0.25 and 0.26, I managed to crack it after a little bit of debugging. The differences mainly reside in the fact that the importer doesn't read the `id` column and that the name is read from the resource's manifest. It's all in [this commit](https://github.com/digidemlab/decidim-module-budgets_paper_ballots/commit/992dd118685d29224bbfe490897b5924f16c6429).
